### PR TITLE
File conversion from txt to csv and json

### PR DESCRIPTION
**Pull request by nrich20**
Created a zarrstore.py file that is similar to the asfstore.py file except it saves the data in the zarr file format.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,10 @@ conda create -n noisepy python=3.8 pip
 conda activate noisepy
 conda install -c conda-forge openmpi
 pip install noisepy-seis
+conda create -n noisepy python=3.8 pip
```
Comment (carlosgjs) [inline README.md:28]: bad merge?

File: pyproject.toml [hunk 2]
```diff
@@ -10,6 +10,7 @@ readme = "README.md"
 license = {file= "LICENSE"}
 requires-python = ">=3.8"
 authors = [
+    { email = "mdenolle@uw.edu", name = "Marine Denolle" },
```
Comment (carlosgjs) [inline pyproject.toml:13]:
```suggestion
```

File: pyproject.toml [hunk 3]
```diff
@@ -33,6 +34,7 @@ classifiers = [
     "Programming Language :: Python :: 3.8",
 ]
 dependencies = [
+    "DateTimeRange==2.0.0",
```
Comment (carlosgjs) [inline pyproject.toml:37]:
```suggestion
```

File: src/noisepy/seis/zarrstore.py [hunk 4]
```diff
@@ -0,0 +1,154 @@
+import datetime
```
Comment (carlosgjs) [inline src/noisepy/seis/zarrstore.py:1]: @nrich20 , can you move this to a separate PR? All of the above is about the txt->csv/json conversion

**Review by carlosgjs — CHANGES_REQUESTED** [review]
Great progress! Some inline comments.

**Comment by nrich20** [comment]
Due to many merge errors and other conflicts, this branch has become inoperable. Closing this pull request and will reopen this issue on a clean branch.

**Pull request closed without merging**